17_Family Tree.py

```diff
@@ -27,7 +26,16 @@
 # does not matter and duplicates will be ignored.
 
 def ancestors(genealogy, person):
-    # your code here
+
+    family=[]
+
+    if person not in genealogy:
+        return family
+    else:
+        for parent in genealogy[person]:
+            family.append(parent)
+            family+=ancestors(genealogy,parent)

```

Comment:
`family+=ancestors(genealogy,parent)`
==>
`family = family + ancestors(genealogy,parent)`
It will create a new list from elements of the two lists but
`family.extent( ancestors(genealogy,parent) )` will insert new items in the existing list.

Reply:
duly noted. thanks